packages.ini

```diff
@@ -646,8 +648,6 @@ brew_requires =
 [pytest-sentry==0.1.11]
 [pytest-sentry==0.1.16]
 
-[pytest-timeout==2.1.0]
-
```

Comment:
typically things are not removed unless there is a security issue -- pypi is add-only